lib/cdo/delete_accounts_helper.rb

```diff
@@ -374,6 +374,12 @@ def purge_user_authentications(user)
     user.authentication_options.with_deleted.order(deleted_at: :desc).each(&:really_destroy!)
   end
 
+  def purge_lti_user_identities(user)
```

Comment:
NOTE: This shouldn't be needed, need to look into [this test](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/helpers/delete_accounts_helper_test.rb) to verify the [Paranoia module](https://github.com/rubysherpas/paranoia#paranoia) will delete child model records from associations (as it says it does).